app/src/main/res/layout/fragment_activity.xml

```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
@SimonMaha for the future, it's better to use revert naming: activity_fragment, signup_activity, profile_fragment. When you will make a search on the project you need just type pro...
instead of fragment_pro.